docs/README.md

```diff
@@ -1,9 +1,10 @@
 # Tablesaw Documentation
 
-* [Introductory Tutorial](https://jtablesaw.github.io/tablesaw/gettingstarted) If you haven't used Tablesaw, or are having trouble 'getting it', you should definitely start here. It covers most everything you need to begin using it productively. 
-* [User Guide](https://jtablesaw.github.io/tablesaw/userguide/toc)
+* [The Tornado Tutorial](https://jtablesaw.github.io/tablesaw/tutorial) A short tutorial using Tablesaw to analyze a Tornado dataset.
+* [Getting Started](https://jtablesaw.github.io/tablesaw/gettingstarted)  If you haven't used Tablesaw, or are having trouble 'getting it', you should start here. It covers most everything you need to begin using it productively. 
```

Comment:
I would think "Getting Started" should be the first bullet if it's really how we want people to get started